### PR TITLE
Move aws skd pkgs to peer deps

### PR DESCRIPTION
**Pull request by Maksym-Haiovets**

### CODE REVIEW

**Review by vladgolubev — APPROVED** [review]

File: readme.md [hunk 1]
```diff
@@ -10,7 +10,7 @@
 $ yarn add @shelf/jest-dynamodb --dev
 ```
 
-Make sure `java` runtime available for running [DynamoDBLocal.jar](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/DynamoDBLocal.DownloadingAndRunning.html)
+Make sure `aws-sdk` is installed as a peer dependency. And `java` runtime available for running [DynamoDBLocal.jar](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/DynamoDBLocal.DownloadingAndRunning.html)
```
Comment (vladgolubev) [inline readme.md:13]: Not `aws-sdk` but 3 `@aws-sdk` packages

**Review by vladgolubev — CHANGES_REQUESTED** [review]
.

File: readme.md [hunk 2]
```diff
@@ -10,7 +10,7 @@
 $ yarn add @shelf/jest-dynamodb --dev
 ```
 
-Make sure `aws-sdk` is installed as a peer dependency. And `java` runtime available for running [DynamoDBLocal.jar](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/DynamoDBLocal.DownloadingAndRunning.html)
+Make sure 3 `@aws-sdk` packages is installed as a peer dependency. And `java` runtime available for running [DynamoDBLocal.jar](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/DynamoDBLocal.DownloadingAndRunning.html)
```
Comment (vladgolubev) [inline readme.md]: I think we don't need peer deps here, as `jest-dynamodb` is always a devDep.
Don't you think so?

Comment (KnupMan) [reply readme.md]: I think we don't need it, sorry for misinforming you.

**Pull request closed without merging**